src/libs/Url.js

```diff
@@ -10,23 +10,7 @@ function addTrailingForwardSlash(url) {
     return url;
 }
 
-/**
- * Add / to the beginning and end of any URL if not present
- * @param {String} url
- * @returns {String}
- */
-function wrapWithForwardSlash(url) {
-    const newUrl = addTrailingForwardSlash(url);
-    if (newUrl.startsWith('/')) {
-        return newUrl;
-    }
-
-    return `/${newUrl}`;
-}
```

Comment:
I don't know why I did this to begin with. All the routes start without a slash.